pkg/daemon/update.go

```diff
@@ -437,39 +437,7 @@ func calculatePostConfigChangeActionFromFileDiffs(oldIgnConfig, newIgnConfig ign
 		"/etc/containers/registries.conf",
 	}
 
-	oldFileSet := make(map[string]ign3types.File)
-	for _, f := range oldIgnConfig.Storage.Files {
-		oldFileSet[f.Path] = f
-	}
-	newFileSet := make(map[string]ign3types.File)
-	for _, f := range newIgnConfig.Storage.Files {
-		newFileSet[f.Path] = f
-	}
-	diffFileSet := []string{}
-
-	// First check if any files were removed
-	for path := range oldFileSet {
-		_, ok := newFileSet[path]
-		if !ok {
-			// debug: remove
-			glog.Infof("File diff: %v was deleted", path)
-			diffFileSet = append(diffFileSet, path)
-		}
-	}
-
-	// Now check if any files were added/changed
-	for path, newFile := range newFileSet {
-		oldFile, ok := oldFileSet[path]
-		if !ok {
-			// debug: remove
-			glog.Infof("File diff: %v was added", path)
-			diffFileSet = append(diffFileSet, path)
-		} else if !reflect.DeepEqual(oldFile, newFile) {
-			// debug: remove
-			glog.Infof("File diff: detected change to %v", newFile.Path)
-			diffFileSet = append(diffFileSet, path)
-		}
-	}
+	diffFileSet := ctrlcommon.CalculateConfigFileDiffs(&oldIgnConfig, &newIgnConfig)
```

Comment:
@mkenigs Looks like John's PR is already refactoring fileDiff list in a separate function (related to https://github.com/openshift/machine-config-operator/pull/2851)

Reply:
@john Logging about changed file will make more sense here.

Reply:
What's the best way to handle that? Should I wait for my PR until this one lands?

Reply:
@mkenigs Ah no, It's fine to continue with your code. Noone is blocked on anyone, tagged you to just for FYI. Whichever PR get merged later may need to fix merge conflict.